.github/workflows/ci.yml

```diff
@@ -218,6 +218,7 @@ jobs:
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
       - name: E2E tests
+        continue-on-error: true
```

Comment:
do we want the PR to be all green even if e2e tests fail?
and to trigger deploy on e2e failure?

Reply:
Was changed to:
if: always()

Reply:
I think we don't need anything there
if: always() will make it run even if previous steps fail I think, e.g. docker build

Reply:
oh, nevermind
if: always() in artifacts is ok
I thought you are putting this in the e2e step